ovs/lib/alba.py

```diff
@@ -60,74 +63,31 @@ class AlbaController(object):
     NSM_PLUGIN = 'nsm_host_plugin'
     ALBA_VERSION_GET = 'alba=`alba version --terse`'
     ARAKOON_PLUGIN_DIR = '/usr/lib/alba'
+    NS_MGR_SERVICE_TYPE = ServiceTypeList.get_by_name(ServiceType.SERVICE_TYPES.NS_MGR)
```

Comment:
Let's not make them global, as this will block code imports if the DAL isn't running.